glossary.yml

```diff
@@ -28,6 +36,10 @@
     term: "afgooiware"
     def: >
       Sagteware wat nie meer onderhou word nie.
+  fr:
+    term: "abandonware"
+    def: >
+      Un logiciel qui n'est plus maintenue.
```

Comment:
Small grammatical error. Should be "Un logiciel qui n'est plus maintenu." (without trailing 'e')

Reply:
Ah yes ! sorry.